openbb_terminal/stocks/fundamental_analysis/fmp_model.py

```diff
@@ -585,3 +585,58 @@ def clean_metrics_df(data: pd.DataFrame, num: int, mask: bool = True) -> pd.Data
     )
 
     return data
+
+
+@log_start_end(log=logger)
+@check_api_key(["API_KEY_FINANCIALMODELINGPREP"])
+def get_filings(
+    pages: int = 1,
+) -> pd.DataFrame:
+    """Get SEC Filings RSS feed, disseminated by FMP
+
+    Parameters
+    ----------
+    pages: range = 1
+        The range of most-rececnt pages to get entries from (1000 per page; maximum of 30 pages)
+
+    Returns
+    -------
+    pd.DataFrame
+        Dataframe of results
+    """
```

Comment:
Since you added this to the sdk, you should add the example usage one would see in the help( fn )

Reply:
Done